controllers/managedocs_controller.go

```diff
@@ -704,6 +709,9 @@ func (r *ManagedOCSReconciler) generateAlertmanagerConfig(pagerdutyServiceKey st
 	alertmanagerConfig.Receivers[0].Name = "pagerduty"
 	alertmanagerConfig.Receivers[0].PagerdutyConfigs = []PagerDutyConfig{{}}
 	alertmanagerConfig.Receivers[0].PagerdutyConfigs[0].ServiceKey = pagerdutyServiceKey
+	alertmanagerConfig.Receivers[0].PagerdutyConfigs[0].SendResolved = false
```

Comment:
What does sendResolved accomplish?

Reply:
I thought it configured if the alert is sent marked as resolved, but  from checking the documentation I see that it actually dictates "Whether or not to notify about resolved alerts." Since it isn't relevant to this PR, I took it out.